src/components/table.tsx

```diff
@@ -71,7 +71,7 @@ const sortButtonStyle = css`
 export interface Column<CellType, RowType> {
   /** Column name, used in title row */
   title: string
-  renderCell: (cellValue: CellType) => JSX.Element
+  renderCell: (cellValue: CellType, cellIndex: number) => JSX.Element
```

Comment:
Should the 2nd parameter be called `rowIndex`?

Reply:
ah, yes